utils/config.go

```diff
@@ -414,13 +414,17 @@ func LoadGlobalConfig(fileName string) *model.Config {
 	SetDefaultRolesBasedOnConfig()
 	SetSiteURL(*Cfg.ServiceSettings.SiteURL)
 
-	for _, listener := range cfgListeners {
-		listener(&oldConfig, config)
-	}
+	InvokeGlobalConfigListeners(&oldConfig, config)
 
 	return config
 }
 
+func InvokeGlobalConfigListeners(old, current *model.Config) {
```

Comment:
Of course I'll be aiming to factor this out soon after finishing my crusade to eliminate global config references (which is someone nearing its end).